alembic/versions/bc10fd684e42_remove_work_on_behalf_of_organization_.py

```diff
@@ -0,0 +1,26 @@
+"""remove work_on_behalf_of_organization from Users table
+
+Revision ID: bc10fd684e42
+Revises: bd67c88713b8
+Create Date: 2021-08-31 19:24:43.751675
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "bc10fd684e42"
+down_revision = "bd67c88713b8"
```

Comment:
As mentioned above - down revision should be ef627578319a since an additional alembic version was added to dev